Sources/MapLibreSwiftUI/Models/MapCamera/MapViewCamera.swift

```diff
@@ -117,4 +117,17 @@ public struct MapViewCamera: Hashable {
     }
     
     // TODO: Create init methods for other camera states once supporting materials are understood (e.g. BoundingBox)
+	
+	public static func boundingBox(northeast: CLLocationCoordinate2D,
```

Comment:
This looks good, only nit here is doc string & spacing (mainly our problem since we don't have swiftlint configured yet 😅)

Reply:
Sweet, will take a look. Haven't really made any decisions on this yet other than noting the fact we need something.